javascripts/discourse/components/tag-game-card-contents.gjs

```diff
@@ -39,20 +52,60 @@ export default class TagGameCardContents extends Component {
   async fetchTagData() {
     const { data } = await graphql(TAG_QUERY, { slug: this.args.data.tag });
 
-    this.tagData = data.product[0];
+    this.priceData = data.product_price[0] || null;
+    this.tagData = this.priceData.product || null;
 
     if (!this.tagData) {
       this.router.transitionTo("tag.show", this.args.data.tag);
     }
+  }
 
-    // TODO remove this
-    this.tagDataJson = JSON.stringify(this.tagData, null, 2);
+  get tagDataJson() {
+    return JSON.stringify(this.tagData);
   }
 
+  get redirectUrl() {
+    return `${settings.main_site_url}/item/${this.args.data.tag}`;
+  }
+
+  get thumbnail_url() {
+    return this.tagData.thumbnail_url
+      ? this.tagData.thumbnail_url
+      : "/images/no_image.png";
+  }
   <template>
     {{#if this.tagData}}
-      <div class="tag-game-card-contents">
-        {{this.tagDataJson}}
+      <div class="tag-game-card-contents game-tag-card">
+        <div class="flex w-full flex-col">
+          <div class="flex">
+            <a href={{this.redirectUrl}}>
+              <div
+                class="game-tag-card__image"
+                style="background-image: url('{{this.thumbnail_url}}');"
+              />
+            </a>
+            <div class="game-tag-card__body">
+              <div class="px-2 pt-2">
+                <a
+                  class="game-tag-card__title"
+                  title={{this.tagData.name}}
+                  href={{this.redirectUrl}}
+                >
+                  {{this.tagData.name}}
+                </a>
+                <TagGameCardInfo @tagData={{this.tagData}} />
+              </div>
+              <div class="px-2 pt-2" style="display: inline-flex;">
```

Comment:
I think we can avoid setting `style` here by creating a named class (and include `px-2 pt-2` there)

Reply:
Yes, I named the class as .custom-container. Any other suggestions?